src/Icons/src/DependencyInjection/UXIconsExtension.php

```diff
@@ -43,6 +43,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('Default attributes to add to all icons.')
                     ->defaultValue(['fill' => 'currentColor'])
                 ->end()
+                ->arrayNode('renderer')
+                    ->info('Options for the icon renderer.')
+                    ->children()
+                        ->booleanNode('auto_aria_hidden')
+                            ->info('Automatically set aria-hidden="true" when no label or title is provided.')
+                            ->defaultTrue()
+                        ->end()
+                    ->end()
+                ->end()
```

Comment:
I don't think we need to add a configuration here. I think document the behaviour should be enough